hollow/src/main/java/com/netflix/hollow/core/write/HollowWriteStateEngine.java

```diff
@@ -192,8 +192,8 @@ public void run() {
             }
 
             executor.awaitSuccessfulCompletion();
-        } catch(Throwable t) {
-            throw new RuntimeException(t);
+        } catch(Exception ex) {
+            throw new RuntimeException(ex);
```

Comment:
We've started replacing rethrowing as `HollowException` or a subclass of `HollowException` just to make forensics a little clearer. We change them as we come across them while making other related changes.